src/meltano/cli/__init__.py

```diff
@@ -59,6 +61,28 @@
 
 logger = logging.getLogger(__name__)
 
+troubleshooting_message = """\
+Need help fixing this problem? Visit http://melta.no/ for troubleshooting steps or to
+join our friendly slack community.
+"""
+
+
+def handle_meltano_error(error: MeltanoError):
+    """Handle a MeltanoError.
+
+    Args:
+        error: The error to handle.
+
+    Raises:
+        CliError: always.
+    """
+    if error.instruction:
+        message = f"{error.reason}. {error.instruction}."
+    else:
+        message = error.reason
+
+    raise CliError(message) from error
```

Comment:
This suggestion avoids inconsistent periods.

```suggestion
    raise CliError(
        f"{error.reason}. {error.instruction}."
        if error.instruction
        else f"{error.reason}."
    ) from error
```